packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -146,6 +146,9 @@ export default class InteropService_Exporter_Md extends InteropService_Exporter_
 			fileName = friendlySafeFilename(resource.title, null, true);
 		}
 
+		// Markdown link destination should not contain space characters
+		fileName = fileName.replace(' ', '_');
```

Comment:
```suggestion
		fileName = fileName.replace(/\s/g, '_');
```
Let's make sure all spaces are being replaced! Notice that
```
% 'foo bar baz.txt'.replace(' ', '_')
`foo_bar baz.txt`
% 'foo bar baz.txt'.replace(/\s/, '_')
`foo_bar baz.txt`
% 'foo bar baz.txt'.replace(/\s/g, '_')
`foo_bar_baz.txt`
%  
```

See the [MDN documentation for `replace` for more information.

Reply:
Nice catch! I updated it.